subprojects/core/src/main/java/org/gradle/execution/plan/QueryableExecutionPlan.java

```diff
@@ -97,6 +97,11 @@ public TaskNode getNode(Task task) {
      * An immutable snapshot of the set of scheduled nodes.
      */
     interface ScheduledNodes {
-        void visitNodes(Consumer<List<Node>> visitor);
+        /**
+         * Invokes the consumer with the list of scheduled nodes and the set of entry nodes. Entry nodes are a subset of scheduled nodes.
```

Comment:
🤔 _Entry nodes are a subset of scheduled nodes_? That contradicts the implementation.

Reply:
> 🤔 _Entry nodes are a subset of scheduled nodes_? That contradicts the implementation.

Oh, here it is. I thought I removed this sentence after I discovered this doesn't always hold.